docs/concepts/function-components/pre-defined-hooks.md

```diff
@@ -197,15 +200,16 @@ The destructor can be used to clean up the effects introduced and it can take ow
 fn effect() -> Html {
     let (counter, set_counter) = use_state(|| 0);
 
-    let counter_one = counter.clone();
-    use_effect(move || {
-        // Make a call to DOM API after component is rendered
-        yew::utils::document().set_title(&format!("You clicked {} times", counter_one));
-
-        // Perform the cleanup
-        || yew::utils::document().set_title(&format!("You clicked 0 times"))
-    });
+    {
+        let counter = counter.clone();
+        use_effect(move || {
+            // Make a call to DOM API after component is rendered
+            yew::utils::document().set_title(&format!("You clicked {} times", counter_one));
```

Comment:
You forgot to change it here:
```suggestion
            yew::utils::document().set_title(&format!("You clicked {} times", counter));
```

Reply:
Wanted to adjust this myself real quick before merging this but it looks like you didn't give me that permission :)

Reply:
Huh I usually give that permissions. Not sure what happened here.

Reply:
Maybe I was just too tired and messed up :smile: